include/envoy/network/dns.h

```diff
@@ -24,6 +24,8 @@ class ActiveDnsQuery {
   virtual void cancel() PURE;
 };
 
+enum class DnsLookupFamily { v4_only, v6_only, fallback };
```

Comment:
This came up in another PR recently: https://github.com/lyft/envoy/pull/989/files/431c909dcd7d70686cc9986c9ef019bef8942ae2#r117533920

I think we are going with all-caps. Feel free to create a https://github.com/lyft/envoy/blob/master/STYLE.md to codify.